### DISCUSSION TITLE
when BetterPlayer start playing.  how can i change BetterPlayerConfiguration

### DISCUSSION THREAD

**Discussion by SinoMiles** [jhomlala/betterplayer#634] [Q&A]
BetterPlayerConfiguration(
        fit: BoxFit.fitWidth}
    betterPlayerController = BetterPlayerController(betterPlayerConfiguration);
when better player start playing. I wanna change BoxFit.fitWidth to BoxFit.contain,how can i do this? any help!
@jhomlala

**Comment by jhomlala** [comment]
You can't change `BetterPlayerConfiguration` once you've build `BetterPlayerController` instance.

**Reply by SinoMiles** [reply]
some video file is very old, we need to set the video size to BoxFit.fitWidth.can you support this feature

**Reply by jhomlala** [reply]
No I won't. This will create a lot of possibilities which may break Better Player. Please set proper config before starting that video.
